lockbox-ios/View/SettingsWebView.swift

```diff
@@ -0,0 +1,38 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import WebKit
+import UIKit
+
+class SettingsWebView: UIViewController {
```

Comment:
nit: those extra `s`s sneak up on you.... should just be `SettingWebView`. please change the file name as well when you change this.